graylog2-web-interface/src/pages/StreamEditPage.jsx

```diff
@@ -30,6 +31,19 @@ const StreamEditPage = React.createClass({
       return <Spinner />;
     }
 
+    let content = <StreamRulesEditor currentUser={this.state.currentUser} streamId={this.props.params.streamId}
```

Comment:
Minor nitpick: this breaks the [`react/wrap-multilines`](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-wrap-multilines.md) style rule.